test/src/main/scala/scalarules/test/fetch_sources/BUILD

```diff
@@ -0,0 +1,8 @@
+load("//scala:scala.bzl", "scala_binary")
+
+scala_binary(
```

Comment:
Nit: this can be a scala library right? Will be simpler. Binary might cause someone to think it’s actually needed and not understand why.